synedrion/src/tools/sss.rs

```diff
@@ -83,31 +99,42 @@ impl Polynomial {
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub(crate) struct PublicPolynomial(Vec<Point>);
+#[serde(bound(deserialize = "for<'x> P: Deserialize<'x>"))]
```

Comment:
Strange that you need an explicit bound for `Deserialize` but not for `Serialize`

Reply:
Agreed. See other comment.